### PR TITLE
deploy: make use of latest attacher release v3.5.0

### PR DESCRIPTION
**Pull request by humblec**
attacher sidecar has a new release:
https://github.com/kubernetes-csi/external-attacher/releases/tag/v3.5.0

Signed-off-by: Humble Chirammal <hchiramm@redhat.com>

### CODE REVIEW

**Comment by humblec** [comment]
the attacher image still in syncing phase in the registry.. will remove the `WIP` once its synced.

**Comment by humblec** [comment]
@Mergifyio rebase

**Comment by humblec** [comment]
> the attacher image still in syncing phase in the registry.. will remove the `WIP` once its synced.

Available now :+1:

File: charts/ceph-csi-rbd/values.yaml [hunk 1]
```diff
@@ -215,7 +215,7 @@ provisioner:
     enabled: true
     image:
       repository: registry.k8s.io/sig-storage/csi-attacher
-      tag: v3.4.0
+      tag: v3.5.0
```
Comment (Madhu-1) [inline charts/ceph-csi-rbd/values.yaml:218]: update helm doc for this one.

**Review by Madhu-1 — CHANGES_REQUESTED** [review]

File: charts/ceph-csi-rbd/values.yaml (hunk 1 above)
Comment (humblec) [reply charts/ceph-csi-rbd/values.yaml:218]: Done.

**Comment by humblec** [comment]
@Mergifyio rebase

**Review by nixpanic — DISMISSED** [review]

**Review by yati1998 — DISMISSED** [review]

**Comment by humblec** [comment]
@Madhu-1 rebase to devel, ptal.. thanks.

**Review by Madhu-1 — APPROVED** [review]

**Review by yati1998 — APPROVED** [review]

**Comment by nixpanic** [comment]
@mergifyio rebase

CI jobs seem to be stuck, and an other PR was merged recently (I think)

**Comment by nixpanic** [comment]
@mergifyio rebase

An other PR was merged, no need to keep retrying before rebasing.

**Pull request merged**